pkg/asset/installconfig/powervs/metadata.go

```diff
@@ -2,7 +2,7 @@ package powervs
 
 import (
 	"context"
-	"fmt"
+	"github.com/openshift/installer/pkg/destroy/powervs"
```

Comment:
nit: move this to its own import block

Reply:
I believe a rewrite removed that import.